data-syncs/supported-data-sync-sources/rest-api.md

```diff
@@ -32,7 +32,15 @@ The following table outlines the mandatory and optional parameters you will find
+|       Parameter        |                                                                                               Description                                                                                                |                                              Example                                              |
+| ---------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| Source                 | Mandatory. Select your source from the drop down menu.                                                                                                                                                   | REST API                                                                                          |
+| HTTP Method            | Mandatory.                                                                                                                                                                                               | This will be either GET or POST.                                                                  |
+| API Response Format    | Mandatory. Use this field to specify a response format of the endpoint. Currently, the Connections UI only supports JSON responses.                                                                      | JSON                                                                                              |
+| Records Root JSONPath  | Mandatory. Specify the JSON path for the results. If the JSON response top-element is an object, use `$.data` to return the parent object of the response, which will contain the array. | `$.data`, `$`                                                                                     |
+| Path to Iterate        | The path of the parameter to iterate. Use to flatten JSON objects.                                                                                                                                       |                                                                                                   |
+| API Endpoint URL       | Mandatory. API endpoint, including URL parameters like API key                                                                                                                                           | https://www.quandl.com/api/v3/datatables/CLS/IDHP?fx_business_date=@date&#x26;amp;api_key=API_KEY |
```

Comment:
What's going on with the URL here?

Does it have to be escaped? Can it not be `?fx_business_date=2024-01-01&api_key=@API_KEY` or something?